### ISSUE TITLE
Failing unit test

### ISSUE THREAD

**Issue by peterdesmet** [inbo/crow#188]
Running `npm run test:unit` results in one failed test:

```shell
> crow-vue-prototype@0.1.0 test:unit
> vue-cli-service test:unit

ts-jest[versions] (WARN) Version 4.9.5 of typescript installed has not been tested with ts-jest. If you're experiencing issues, consider using a supported version (>=2.7.0 <4.0.0). Please do not report issues in ts-jest if you are using unsupported versions.
ts-jest[versions] (WARN) Version 4.9.5 of typescript installed has not been tested with ts-jest. If you're experiencing issues, consider using a supported version (>=2.7.0 <4.0.0). Please do not report issues in ts-jest if you are using unsupported versions.
ts-jest[versions] (WARN) Version 4.9.5 of typescript installed has not been tested with ts-jest. If you're experiencing issues, consider using a supported version (>=2.7.0 <4.0.0). Please do not report issues in ts-jest if you are using unsupported versions.
 PASS  tests/unit/TimelineChart.spec.js
 PASS  tests/unit/VPChart.spec.ts
 FAIL  tests/unit/crow.spec.ts
  ● Test suite failed to run

    TypeScript diagnostics (customize using `[jest-config].globals.ts-jest.diagnostics` option):
    tests/unit/crow.spec.ts:116:20 - error TS2339: Property 'loadData' does not exist on type 'ExtractComputedReturns<{}> & Vue<Record<string, any>, Record<string, any>, never, never, (event: string, ...args: any[]) => Vue<Record<string, any>, Record<...>, never, never, ...>> & ShallowUnwrapRef<...>'.

    116   await wrapper.vm.loadData()
                           ~~~~~~~~
    tests/unit/crow.spec.ts:118:32 - error TS2339: Property 'radarVpts' does not exist on type 'ExtractComputedReturns<{}> & Vue<Record<string, any>, Record<string, any>, never, never, (event: string, ...args: any[]) => Vue<Record<string, any>, Record<...>, never, never, ...>> & ShallowUnwrapRef<...>'.

    118   let vptsAllRows = wrapper.vm.radarVpts
                                       ~~~~~~~~~
    tests/unit/crow.spec.ts:121:33 - error TS2339: Property 'radarVpts' does not exist on type 'ExtractComputedReturns<{}> & Vue<Record<string, any>, Record<string, any>, never, never, (event: string, ...args: any[]) => Vue<Record<string, any>, Record<...>, never, never, ...>> & ShallowUnwrapRef<...>'.

    121   expect(Object.keys(wrapper.vm.radarVpts).map(x => parseInt(x))).toEqual(expect.arrayContaining(
                                        ~~~~~~~~~
    tests/unit/crow.spec.ts:124:33 - error TS2339: Property 'radarVpts' does not exist on type 'ExtractComputedReturns<{}> & Vue<Record<string, any>, Record<string, any>, never, never, (event: string, ...args: any[]) => Vue<Record<string, any>, Record<...>, never, never, ...>> & ShallowUnwrapRef<...>'.

    124   expect(Object.keys(wrapper.vm.radarVpts).map(x => parseInt(x))).toEqual(expect.not.arrayContaining(
                                        ~~~~~~~~~
    tests/unit/crow.spec.ts:151:29 - error TS2339: Property 'integratedProfiles' does not exist on type 'ExtractComputedReturns<{}> & Vue<Record<string, any>, Record<string, any>, never, never, (event: string, ...args: any[]) => Vue<Record<string, any>, Record<...>, never, never, ...>> & ShallowUnwrapRef<...>'.

    151     let jsData = wrapper.vm.integratedProfiles.filter((x: VPIEntry) => x.moment.isSame(moment.utc(timeStamp)))
                                    ~~~~~~~~~~~~~~~~~~
    tests/unit/crow.spec.ts:179:20 - error TS2339: Property 'loadData' does not exist on type 'ExtractComputedReturns<{}> & Vue<Record<string, any>, Record<string, any>, never, never, (event: string, ...args: any[]) => Vue<Record<string, any>, Record<...>, never, never, ...>> & ShallowUnwrapRef<...>'.

    179   await wrapper.vm.loadData()
                           ~~~~~~~~
    tests/unit/crow.spec.ts:181:32 - error TS2339: Property 'radarVpts' does not exist on type 'ExtractComputedReturns<{}> & Vue<Record<string, any>, Record<string, any>, never, never, (event: string, ...args: any[]) => Vue<Record<string, any>, Record<...>, never, never, ...>> & ShallowUnwrapRef<...>'.

    181   let vptsAllRows = wrapper.vm.radarVpts
                                       ~~~~~~~~~
    tests/unit/crow.spec.ts:184:33 - error TS2339: Property 'radarVpts' does not exist on type 'ExtractComputedReturns<{}> & Vue<Record<string, any>, Record<string, any>, never, never, (event: string, ...args: any[]) => Vue<Record<string, any>, Record<...>, never, never, ...>> & ShallowUnwrapRef<...>'.

    184   expect(Object.keys(wrapper.vm.radarVpts).map(x => parseInt(x))).toEqual(expect.arrayContaining(
                                        ~~~~~~~~~
    tests/unit/crow.spec.ts:187:33 - error TS2339: Property 'radarVpts' does not exist on type 'ExtractComputedReturns<{}> & Vue<Record<string, any>, Record<string, any>, never, never, (event: string, ...args: any[]) => Vue<Record<string, any>, Record<...>, never, never, ...>> & ShallowUnwrapRef<...>'.

    187   expect(Object.keys(wrapper.vm.radarVpts).map(x => parseInt(x))).toEqual(expect.not.arrayContaining(
                                        ~~~~~~~~~

Test Suites: 1 failed, 2 passed, 3 total
Tests:       5 passed, 5 total
Snapshots:   0 total
Time:        2.022s, estimated 3s
Ran all test suites.
```

The issue seems to be related to the (new?) `radarVpts` property.

**Comment by niconoe** [comment]
PR here: https://github.com/inbo/crow/pull/189

**Comment by peterdesmet** [comment]
Fixed in #187
